processor/deltatocumulativeprocessor/internal/data/add.go

```diff
@@ -24,9 +25,44 @@ func (dp Number) Add(in Number) Number {
 	return dp
 }
 
-// nolint
 func (dp Histogram) Add(in Histogram) Histogram {
-	panic("todo")
+	// bounds different: no way to merge, so reset observation to new boundaries
+	if !pslice.Equal(dp.ExplicitBounds(), in.ExplicitBounds()) {
+		in.MoveTo(dp.HistogramDataPoint)
```

Comment:
Should we add a metric for how often this occurs?

Reply:
sgtm, but will do later as we don't have a good way of accessing a `Meter` here without refactoring the `telemetry` package